core/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>stargate</artifactId>
+        <groupId>io.stargate</groupId>
+        <version>0.0.10-SNAPSHOT</version>
+    </parent>
+    <groupId>io.stargate.core</groupId>
+    <artifactId>core</artifactId>
+    <properties>
+        <dropwizard.version>2.0.10</dropwizard.version>
+    </properties>
+    <dependencies>
+        <dependency>
```

Comment:
I am not sure if we should include `slf4j` in the core? What if other modules that are using it provide dependencies to the logging library? All sub-module will need to exclude any logging library, right?

Reply:
Without this dependency, core module won't activate: 

```
Starting bundle io.stargate.core
Unexpected error: Activator start error in bundle io.stargate.core [8].
org.osgi.framework.BundleException: Activator start error in bundle io.stargate.core [8].
	at org.apache.felix.framework.Felix.activateBundle(Felix.java:2452)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2308)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:984)
	at io.stargate.starter.Starter.start(Starter.java:368)
	at io.stargate.starter.Starter.main(Starter.java:477)
Caused by: java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
	at io.stargate.core.CoreActivator.<clinit>(CoreActivator.java:13)
```

Reply:
Should the scope be `provided` perhaps? I believe `slf4j` is part of the system class path in our OSGi container (if I'm not mistaken)

Reply:
I'll give it a try.

Reply:
With `provided`, I also get:

```
Starting bundle io.stargate.core
Unexpected error: Activator start error in bundle io.stargate.core [8].
org.osgi.framework.BundleException: Activator start error in bundle io.stargate.core [8].
	at org.apache.felix.framework.Felix.activateBundle(Felix.java:2452)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2308)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:984)
	at io.stargate.starter.Starter.start(Starter.java:368)
	at io.stargate.starter.Starter.main(Starter.java:477)
Caused by: java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
	at io.stargate.core.CoreActivator.<clinit>(CoreActivator.java:13)
```

Reply:
hmmm, interesting, I guess we can leave it for now, IIRC we have a dedicated issue for improving logging